### ISSUE TITLE
Drop tests are flaky

### ISSUE THREAD

**Issue by lucas-clemente** [lucas-clemente/quic-go#243]
After ~50 attempts:

```
• Failure [4.008 seconds]
Drop Proxy
/Users/lucas/src/go/src/github.com/lucas-clemente/quic-go/integrationtests/drop_test.go:80
  with quic version 32
  /Users/lucas/src/go/src/github.com/lucas-clemente/quic-go/integrationtests/drop_test.go:78
    dropping every 4th packet after the crypto handshake
    /Users/lucas/src/go/src/github.com/lucas-clemente/quic-go/integrationtests/drop_test.go:77
      gets a file when many outgoing packets are dropped [It]
      /Users/lucas/src/go/src/github.com/lucas-clemente/quic-go/integrationtests/drop_test.go:72

      Timed out after 4.000s.
      Expected process to exit.  It did not.

      /Users/lucas/src/go/src/github.com/lucas-clemente/quic-go/integrationtests/drop_test.go:49
------------------------------
```

**Comment by marten-seemann** [comment]
Closed the wrong issue. Force-pushing the fix and reopening.

**Comment by marten-seemann** [comment]
It seems like the fix for #250 just made this issue worse.
Maybe it's related to #248, but we definitely need to fix this one asap, it's becoming really annoying.
